.eslintrc

```
@@ -11,11 +11,11 @@
         "indent": [2, 4],
         "max-lines-per-function": [2, { "max": 150 }],
         "max-params": [2, 14],
-        "max-statements": [2, 52],
+        "max-statements": [2, 53],
         "multiline-comment-style": 0,
         "no-continue": 1,
         "no-magic-numbers": 0,
         "no-restricted-syntax": [2, "BreakStatement", "DebuggerStatement", "ForInStatement", "LabeledStatement", "WithStatement"],
-        "operator-linebreak": [2, "before"],
+        "operator-linebreak": [2, "before"]
```

trailing commas are fine in eslintrc files, fwiw
```suggestion
        "operator-linebreak": [2, "before"],
```